### ISSUE TITLE
Watch multiple nodes in the trace log

### ISSUE THREAD

**Issue by binji** [floooh/v6502r#9]
It seems like the values of all nodes are tracked in the trace log, but you can only watch one node at a time. It would be nice to be able to display more than just one value here (maybe there's already a way?)

**Comment by floooh** [comment]
Yeah there isn't currently a way to watch multiple custom nodes in the trace log, I was cheaping out a bit on the implementation :) The information is there though, because the tracelog internally stores the entire CPU state, it's just the UI that's lacking.

**Comment by binji** [comment]
> The information is there though, because the tracelog internally stores the entire CPU state

I noticed! It's pretty cool to be able to change the watch values after they've already executed. :-)

I might take a look at adding something simple for this, it was requested by someone playing around with v2a03remix
